### ISSUE TITLE
Comb Filter vs KeyUp during Mono Mode

### ISSUE THREAD

**Issue by MatthiasSeeber** [nonlinear-labs-dev/C15#1460]
When using Mono Mode (Legato can be on or off) and playing Legato-style (with overlapping keys), then releasing a key (that should be ignored) will cause the Comb Filter to fall silent.
The Gate signal seems involved in it, as well as the PolySection KeyUp procedure.

**Comment by MatthiasSeeber** [comment]
This issue is valid for the Test Unit using the current compilate (branch "uffff" at c9ecdfc) due to an improper implementation of the candidate Event-based Gate controlling the Comb Filter. A quick test revealed that the implementation could easily be corrected in order to produce expected behavior.

However, as it already was decided that the Event-based mechanism will not be adopted (due to Key release artifacts), it already is removed in the latest AudioEngine version (branch "NAMM-1.7-Addendum" at 5e3bd37), falling back to the (already properly implemented) Signal-based Gate solution.

In other words, it can't be reproduced anymore and should not occur after merging and installing the latest build.

**Comment by schlenkibus** [comment]
@MatthiasSeeber can this be closed then?
